### DISCUSSION TITLE
useResizeColumns - delay table render until after column resize stops

### DISCUSSION THREAD

**Discussion by wildcat-cs** [TanStack/table#2823] [Q&A]
Has anyone tried to keep the table from re-rendering until after the user releases their mouse button when resizing columns?  

I've seen the column resize interaction on a few other grids where the actual resize of the column (and all the cells) does not occur until after the user releases their mouse button, sometimes the grids will display a vertical line in the table where the column will be placed (but that's just eye candy).  This approach reduces a lot of rendering while the drag is occurring particularly when you have many rows/columns which gives the grid the appearance of having improved performance.  

Just throwing it out there, would this require a new plugin or has anyone got it working with the existing one?

Thanks
Jeff

**Comment by topolanekmartin** [comment]
Any update?
